### ISSUE TITLE
Flexweave Underlay cloak enchant showing as spell power enchant

### ISSUE THREAD

**Issue by Rogumann** [wowsims/wotlk#1982]
**Describe the bug**
When i use the addon warcraft sims exporter, and copy it into my feral simulator https://wowsims.github.io/wotlk/feral_druid/#, my Flexweave Underlay cloak enchant shows as +27 spell power instead of the 23 agility + parachute enchant that it really is. Its just annoying.

**To Reproduce**
1. Equip "Hammerhead Sharkskin Cloak" (have not tested with other cloaks"
2. Use the export data from the addon /wse export
3. Import into feral sim https://wowsims.github.io/wotlk/feral_druid/# and see the incorrect enchant

**Expected behavior**
Expect it to show the correct enchant

**Sim Links and Screenshots**
https://wowsims.github.io/wotlk/feral_druid/#

**Desktop (please complete the following information):**
 - OS: W10
 - Browser Firefox
 - Version 107.0

**Comment by lologarithm** [comment]
Could you paste your import in here -- I want to see if its an issue with the wse exporter or our import logic on the site.

**Comment by Rogumann** [comment]
Of course, its below. This actually happens every single time I import from the addon.

{"talents":"0000000000000000000000000000-503202132322010053120230310511-205503012000000000000000000","glyphs":{"major":["Glyph of Shred","Glyph of Savage Roar","Glyph of Rip"],"minor":["Glyph of Dash","Glyph of the Wild","Glyph of Unburdened Rebirth"]},"class":"druid","race":"NightElf","name":"Pusekatt","gear":{"items":[{"enchant":3817,"gems":[41398,39996],"id":39553},{"id":40065,"gems":[]},{"enchant":3808,"gems":[39996],"id":40494},{"enchant":3859,"gems":[],"id":40721},{"enchant":3252,"gems":[40038,40022],"id":40277},{"enchant":3845,"gems":[],"id":40186},{"enchant":3604,"gems":[],"id":39727},{"id":40205,"gems":[39996]},{"enchant":3823,"gems":[39996,42702],"id":44011},{"enchant":3606,"gems":[],"id":40748},{"id":40074,"gems":[]},{"id":40717,"gems":[]},{"id":37723,"gems":[]},{"id":40684,"gems":[]},{"enchant":3828,"gems":[],"id":39422},null,{"id":40713,"gems":[]}]},"professions":[{"name":"Engineering","level":450},{"name":"Mining","level":450}],"level":80,"spec":"feral","realm":"Giantstalker"}


Med vennlig hilsen
Lasse Pladsen
+47 94 26 77 61
***@***.***
________________________________
Fra: Ben Echols ***@***.***>
Sendt: torsdag 1. desember 2022 03:50
Til: wowsims/wotlk ***@***.***>
Kopi: Rogumann ***@***.***>; Author ***@***.***>
Emne: Re: [wowsims/wotlk] Flexweave Underlay cloak enchant showing as spell power enchant (Issue #1982)


Could you paste your import in here -- I want to see if its an issue with the wse exporter or our import logic on the site.

**Comment by jimmyt857** [comment]
The enchant ID being exported by the addon is `3859`, contained in the block with the cloak `{"enchant":3859,"gems":[],"id":40721}`. 3859 is the effect ID for Springy Archanoweave (https://www.wowhead.com/wotlk/spell=63765/springy-arachnoweave), so the importer is behaving correctly here. The effect ID for flexweave underlay is 3605 (https://www.wowhead.com/wotlk/spell=55002/flexweave-underlay).

This COULD be a problem with the export addon. However, given that nobody has reported this before, the first thing I would suggest is making sure your cloak actually has the enchant you think it has. :)

**Comment by Rogumann** [comment]
Might bug report it to the export addon then (i have no idea if you are the same guys), but yes that is in fact the enchant im using. If not i would surely have been flamed by my GM.

Med vennlig hilsen
Lasse Pladsen
+47 94 26 77 61
***@***.***
________________________________
Fra: James Tanner ***@***.***>
Sendt: lørdag 3. desember 2022 17:16
Til: wowsims/wotlk ***@***.***>
Kopi: Rogumann ***@***.***>; Author ***@***.***>
Emne: Re: [wowsims/wotlk] Flexweave Underlay cloak enchant showing as spell power enchant (Issue #1982)


The enchant ID being exported by the addon is 3859, contained in the block with the cloak {"enchant":3859,"gems":[],"id":40721}. 3859 is the effect ID for Springy Archanoweave (https://www.wowhead.com/wotlk/spell=63765/springy-arachnoweave), so the importer is behaving correctly here. The effect ID for flexweave underlay is 3605 (https://www.wowhead.com/wotlk/spell=55002/flexweave-underlay).

This COULD be a problem with the export addon. However, given that nobody has reported this before, the first thing I would suggest is making sure your cloak actually has the enchant you think it has. :)

**Comment by jimmyt857** [comment]
@generalwrex could you take a look at this then?

**Comment by Rogumann** [comment]
It seems to work correctly when i disable all addons except wse exporter (and bagnon, cause without another addon i couldnt open the exporter for some reason) so it must be some weird interaction with some other addon. I have not tested everyone to see which it could be yet tho...

Med vennlig hilsen
Lasse Pladsen
+47 94 26 77 61
***@***.***
________________________________
Fra: James Tanner ***@***.***>
Sendt: lørdag 3. desember 2022 18:07
Til: wowsims/wotlk ***@***.***>
Kopi: Rogumann ***@***.***>; Author ***@***.***>
Emne: Re: [wowsims/wotlk] Flexweave Underlay cloak enchant showing as spell power enchant (Issue #1982)


@generalwrex<https://github.com/generalwrex> could you take a look at this then?

**Comment by jimmyt857** [comment]
If you don't mind trying to identify the addon, that would help us a lot. You can do it more quickly using a binary search strategy, i.e. disable half your addons, then half of the remaining options, etc

**Comment by Rogumann** [comment]
Ill do my best when i get the time, in the middle of finals period right now so im a bit busy.

Med vennlig hilsen
Lasse Pladsen
+94267761
***@***.***
________________________________

**Comment by jimmyt857** [comment]
It's been quite awhile and we haven't heard of anyone else with this issue, so there's likely something wrong on your end. Make sure your addon is up to date, and feel free to report if you still have problems. Otherwise going to close this for now.
